Dockerfile

```diff
@@ -1,19 +1,14 @@
-FROM quay.io/keboola/aws-cli
-ARG AWS_SECRET_ACCESS_KEY
-ARG AWS_ACCESS_KEY_ID
-ARG AWS_SESSION_TOKEN
-# How to update drivers - https://github.com/keboola/drivers-management
-RUN /usr/bin/aws s3 cp s3://keboola-drivers/snowflake/snowflake-odbc-2.16.10.x86_64.deb /tmp/snowflake-odbc.deb
-RUN /usr/bin/aws s3 cp s3://keboola-drivers/snowflake/snowsql-1.1.68-linux_x86_64.bash /tmp/snowsql-linux_x86_64.bash
-
 FROM php:7.1
 
+ARG SNOWFLAKE_ODBC_VERSION=2.16.10
```

Comment:
Tohle mělo být v samostatném PR :(

Reply:
A mohlo se to mergnout samostatně nebo naopak nebrzdit ten update db-ex-common

Reply:
je to v jiném PR ale vycházel jsem z něj - doplnil jsem to do description PR

Reply:
https://github.com/keboola/db-extractor-snowflake/pull/87 už je mergnuté a tohle tady pořád je. Jakto?

Reply:
není to tím že to původně bylo tady a až potom jsem to dodatečně rozděloval?

Reply:
Bylo to tím, že to nebylo rebasnuté.